test/system/src/test/resources/pravega_withTLS.properties

```diff
@@ -30,10 +30,10 @@ controller.security.pwdAuthHandler.accountsDb.location=/opt/pravega/conf/passwd
 controller.security.auth.delegationToken.signingKey.basis=secret
 autoScale.controller.connect.security.auth.enable=true
 autoScale.security.auth.token.signingKey.basis=secret
-#4GB of cache.
-pravegaservice.cache.size.max=4294967296
+#2GB of cache.
+pravegaservice.cache.size.max=3094967296
```

Comment:
This looks like 3GB (or less), but not 2GB.

Reply:
set it to 3G